CONTRIBUTING.md

```diff
@@ -4,79 +4,56 @@ Thank you for your interest in contributing to Dataverse! We welcome contributio
+You can attach certain files (images, screenshots, logs, etc.) by dragging and dropping, selecting them, or pasting from the clipboard. Files must be one of GitHub's [supported attachment formats] such as png, gif, jpg, txt, pdf, zip, etc. (Pro tip: A file ending in .log can be renamed to .txt so you can upload it.) If there's no easy way to attach your file, please include a URL that points to the file in question.
 
-[supported image format]: https://help.github.com/articles/issue-attachments
-
-## Milestones
-  - **In Review**: milestone for issues that are being reviewed for future releases
-  - **In Design**: milestone for issues that are being designed (FRDs, mockups, technical specifications, etc happen during this milestone). This is a way to see what we are working on for future releases
-  - **Candidate** candidate for [version] means that it is only under consideration. A developer works on issues that have been tagged with a particular milestone before dipping into the corresponding "candidate for" milestone. Usually are "nice to have" features or bug fixes for a release if there is time left after the primary work is done.
+[supported attachment formats]: https://help.github.com/articles/file-attachments-on-issues-and-pull-requests/
 
 ## Documentation
 
-The source for the documentation at http://guides.dataverse.org is in the GitHub repo under the "[doc][]" folder. If you find a typo or inaccuracy or something to clarify, please send us a pull request!
+The source for the documentation at http://guides.dataverse.org is in the GitHub repo under the "[doc][]" folder. If you find a typo or inaccuracy or something to clarify, please send us a pull request! For more on the tools used to write docs, please see the [documentation][] section of the Developer Guide.
+
+[doc]: https://github.com/IQSS/dataverse/tree/develop/doc/sphinx-guides/source
+[documentation]: http://guides.dataverse.org/en/latest/developers/documentation.html
 
 ## Code/Pull Requests
 
 To get started developing code for Dataverse, please read our Developer Guide at http://guides.dataverse.org/en/latest/developers
 
-[doc]: https://github.com/IQSS/dataverse/tree/master/doc/sphinx-guides/source
-
 Before you start coding, please reach out to us either on our [dataverse-community Google Group][], [IRC][], or via support@dataverse.org to make sure the effort is well coordinated and we avoid merge conflicts.
 
 [dataverse-community Google Group]: https://groups.google.com/group/dataverse-community
-[IRC]: http://irclog.iq.harvard.edu/dataverse/today
+[IRC]: http://chat.dataverse.org
 
 If your pull request is not assigned to anyone in a timely manner, please reach out. The assignee is responsible for evaluating the pull request and deciding whether or not to merge it in. Please try to make it easy to merge in pull requests. Tests are great. :)
```

Comment:
I just re-read "The assignee is responsible for evaluating the pull request and deciding whether or not to merge it in" and this is an out-of-date holdover from an old process. I'll update it.